tedge/src/cli/disconnect/cli.rs

```diff
@@ -0,0 +1,38 @@
+use crate::{
+    cli::{
+        connect::{az::AZURE_CONFIG_FILENAME, c8y::C8Y_CONFIG_FILENAME},
+        disconnect::disconnect_bridge::DisconnectBridgeCommand,
+    },
+    command::{BuildCommand, BuildContext, Command},
+};
```

Comment:
I'm not sure to sum up with lots of layers is a good strategy. 
For me, the one below looks easier to read...
```suggestion
use crate::cli::connect::{az::AZURE_CONFIG_FILENAME, c8y::C8Y_CONFIG_FILENAME};
use crate::cli::disconnect::disconnect_bridge::DisconnectBridgeCommand;
use crate::command::{BuildCommand, BuildContext, Command};
```

Reply:
In terms of following certain patterns I followed pattern from config cli where eg:
```
pub use self::{cli::*, commands::*, config_key::*};
```